tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "types": ["node", "jest"],
+    "target": "es5",
```

Comment:
Could you target `ES2019`? See https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping - we run Node 12.x for now